docs/src/main/sphinx/security/user-mapping.rst

```diff
@@ -95,9 +95,9 @@ also maps users in the form ``bob@uk.example.com`` to ``bob_uk``:
 .. literalinclude:: user-mapping.json
     :language: json
 
-Configure this example file to use the :doc:`LDAP </security/ldap>`
-authentication method (which uses the PASSWORD authentication type), as
-follows:
+Set up the preceding example to use the :doc:`LDAP </security/ldap>`
+authentication method (must be set to PASSWORD) by adding the following line
```

Comment:
what must be set to `PASSWORD`? Some people might be able to infer that, but I don't think most will.

It might be helpful to mention the config property name which needs to be set to password or link to relevant section.